projects/index.html

```diff
@@ -255,8 +305,97 @@ <h3>History of Developmental Evolution</h3>
   											</div>
                         </div>
 										</div> -->
-                      <p><strong>Check back later for updated information on our research projects.</strong></p>
-                  </section>
+			<p><strong>Check back later for updated information on our research projects.</strong></p>
+		</section>
+
+		<section>
+
+			{% include section-header.html title="Recent Github Commits" %}
+
+			<table id="#dtBasicExample" style="border-collapse: collapse;">
+
+				<p id="paragraph"></p>
+
+			</table>
+
+
+		</section>
+
+	</div>
+</div>
+
+<script>
+
+	axios.get('https://api.github.com/orgs/diging/events')
+	.then((response) => {
+		data = response.data;
+		console.log(data);
+		
+		for(let i = 0; i < Object.keys(data).length; i++)
+		{
+			
+			if("PushEvent".localeCompare(Object.values(data)[i]["type"]) == 0)
+			{
+
+				let rowAttr = "myTr" + i.toString();
+
+				let row = document.createElement("TR");
+				row.setAttribute("id", rowAttr);
+				document.getElementById("#dtBasicExample").appendChild(row);
+
+				for(let j = 0; j < 4; j++)
+				{
+					let cell = document.createElement("TD");
+
+					switch(j) 
+					{
+						case 0:
+							let repo = Object.values(data)[i]["repo"]["name"];
+							repo = repo.substring(7);
+							cell.appendChild(document.createTextNode(repo));
+							cell.style.width = "155px";
+							document.getElementById(rowAttr).appendChild(cell);
+							break;
+						case 1:
+							let author = Object.values(data)[i]["payload"]["commits"][0]["author"]["name"];
+							cell.appendChild(document.createTextNode(author));
+							document.getElementById(rowAttr).appendChild(cell);
+							break;
+						case 2:
+							let message = Object.values(data)[i]["payload"]["commits"][0]["message"];
+							let asterisk = message.indexOf("\n*");
+							
+							if(asterisk > -1)
+							{
+								message = message.substring(0, asterisk);
+								// document.getElementById("paragraph").innerHTML += message + '<br>';
+								cell.appendChild(document.createTextNode(message));
+							}
+							else
+							{
+								// document.getElementById("paragraph").innerHTML += message + '<br>';
+								cell.appendChild(document.createTextNode(message));
+							}
+							document.getElementById(rowAttr).appendChild(cell);
+							break;
+						case 3:
+							let timeStamp = Object.values(data)[i]["created_at"];
+							timeStamp = timeStamp.substring(0,10) + "\n" + timeStamp.substring(11, 19) + " UTC";
+							cell.appendChild(document.createTextNode(timeStamp));
+							cell.style.width = "120px";
+							document.getElementById(rowAttr).appendChild(cell);
+							break;
+					}
+				}
+
+				// document.getElementById(rowAttr).appendChild(cell);
+				
+			}
+		}
+		
+	}).catch((error) =>{
+		console.log("error");
```

Comment:
there should be some kind error message where the events would be saying something like "We're sorry, but GitHub events could not be loaded."